internal/boxcli/shellenv.go

```diff
@@ -95,11 +95,12 @@ func shellEnvFunc(cmd *cobra.Command, flags shellEnvCmdFlags) (string, error) {
 
 func shellEnvOnlyPathWithoutWrappersCmd() *cobra.Command {
 	command := &cobra.Command{
-		Use:     "only-path-without-wrappers",
-		Hidden:  true,
-		Short:   "[internal] Print shell command that exports the system $PATH without the bin-wrappers paths.",
-		Args:    cobra.ExactArgs(0),
-		PreRunE: ensureNixInstalled,
+		Use:        "only-path-without-wrappers",
+		Deprecated: "This command is deprecated and will be removed after devbox 0.7.0",
```

Comment:
lets confirm this prints to Stderr and not Stdout, otherwise existing bin-wrappers may not work.

That said, I suppose we will re-generate the bin-wrappers due to the Devbox version change when this is released, so may be not a concern either way.

Reply:
Good call. It's actually `OutOrStdErr` which is currently Stderr, but we can't guarantee that. Will remove.